endoreg_db/utils/paths.py

```
@@ -6,37 +6,27 @@
-
-STORAGE_DIR = _resolve_storage_dir()
-STORAGE_DIR.mkdir(parents=True, exist_ok=True)
+import os
+# Alternative approach using env_path helper, deprecated since monorepo setup. Alright for single install, env is always preferred.
+#from endoreg_db.config.env import env_path
+
+STORAGE_DIR = Path(os.getenv("STORAGE_DIR", "storage"))
+
+storage_dir_env = os.getenv("STORAGE_DIR")
+if storage_dir_env is None:
+    raise RuntimeError("STORAGE_DIR environment variable is not set.")
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)